modules/core/src/test/scala/gem/arb/Offset.scala

```diff
@@ -9,13 +9,15 @@ import org.scalacheck._
 import org.scalacheck.Arbitrary._
 
 trait ArbOffset {
-  import ArbAngle._
-
   implicit val arbOffsetP: Arbitrary[Offset.P] =
-    Arbitrary(arbitrary[Angle].map(Offset.P(_)))
+    Arbitrary(
+      Gen.chooseNum(0, 10000).map(mas => Offset.P(Angle.fromMilliarcseconds(mas)))
+    )
```

Comment:
Offsets are stored in the database as signed arcseconds, so small negatives angles should also work.